chainstate-types/src/lib.rs

```diff
@@ -13,7 +13,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-pub mod block_index;
-pub mod height_skip;
-pub mod locator;
 pub mod storage_result;
+
+pub use crate::{
+    block_index::BlockIndex, error::PropertyQueryError, gen_block_index::GenBlockIndex,
```

Comment:
In my opinion it is really inconvenient to always import these types like this:
```
use chainstate_types::{block_index::BlockIndex, gen_block_index::GenBlockIndex};
```
These modules contain only one type each and type names are already contains all needed context, so I see no harm in that.